### DISCUSSION TITLE
Option to hide similar browser engines

### DISCUSSION THREAD

**Discussion by nickserv** [mdn/mdn-community#212] [Browser compatibility data] [answered]
I currently see 15 browsers in [browser-compat-data](https://github.com/mdn/browser-compat-data/tree/main/browsers). However nowadays Internet Explorer is dead and pretty much every import browser/engine is based on Chromium, WebView, or Gecko. I would like to display only major engines (not related engines) since they tend to have similar support, and MDN doesn't include usage data like Can I use. Mobile could possibly be a separate category as well, though I'd like to see iOS Safari by default as it's typically the most restrictive of modern browsers.

For example:
1. Chrome/Chromium (includes V8 but shows Node.js version)
2. Chrome/Chromium Android
3. Safari/WebKit (includes JavaScriptCore but shows Deno version)
4. Safari iOS (shows iOS version)
5. Firefox/Gecko

![image](https://user-images.githubusercontent.com/927220/187069827-02bb3b49-2604-4498-8a73-72019073f620.png)

**Comment by Rumyra** [comment, marked as answer]
Hi @nickmccurdy - we're currently looking to do some research into this area before making any changes to the compat tables. Once that's done we'll share insights and that will drive any modifications we make 👍

**Reply by nickserv** [reply]
Sounds good! Feel free to reply or email if you're looking for feedback.

**Comment by pepelsbey** [comment]
Thank you for starting the discussion. I’m closing this now due to inactivity, but if you think that’s a mistake, feel free to reply. You’re also welcome to join the [MDN Discord](https://developer.mozilla.org/discord) if you’d prefer to chat with us more about it in sync.
